### PR TITLE
Statsd

### PR DESCRIPTION
**Pull request by yoshiotu**
Three new parameters:

BanyanStatsd: true/false
BanyanStatsdAddress: 127.0.0.1:8125
BanyanDDAPIKey: datadog API key (when set, Datadog agent will be installed)

### CODE REVIEW

File: cloudformation/banyan-elastic-access-tier.json [hunk 1]
```diff
@@ -319,6 +335,10 @@
                   "Ref": "AWS::Region"
                 },
                 "\n",
+		"export DD_API_KEY=", { "Ref": "BanyanDDAPIKey" }, "\n",
+		"if [[ -n \"${DD_API_KEY}\" ]]; then\n",
+		"bash -c \"$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)\"\n",
```
Comment (dd76) [inline cloudformation/banyan-elastic-access-tier.json]: is there a reason for bash again within the bash script?

Comment (yoshiotu) [reply cloudformation/banyan-elastic-access-tier.json]: this is the one-line install that Datadog suggests: https://app.datadoghq.com/account/settings?_gl=1*112yu14*_ga*MTc0NjE3NDI0MC4xNjI2ODEwNTA1*_ga_KN80RDFSQK*MTYyNjgxMDUwNC4xLjEuMTYyNjgxMDUxOS4w#agent/aws

Comment (yoshiotu) [reply cloudformation/banyan-elastic-access-tier.json]: But thanks for asking. It turns out that Datadog has a new download URL. I updated the script to use this URL.

Comment (yoshiotu) [reply cloudformation/banyan-elastic-access-tier.json]: I think the idea of the separate bash is that the curl part downloads a bash script, and then you need to run bash in order to execute that downloaded script.

**Review by dd76 — APPROVED** [review]

**Pull request merged**